inc/class_parser.php

```diff
@@ -1458,8 +1458,11 @@ function mycode_parse_video($video, $url)
 
 		$id = htmlspecialchars_uni($id);
```

Comment:
This will lead to double escaping as the `id` is also escaped in the template.

Reply:
Thank you for spotting this, will get rid of it.